docs/features/translation_memory.rst

```diff
@@ -74,4 +78,77 @@ Once populated Pootle will keep Local TM up-to-date.
 Local TM settings can be adjusted in :setting:`POOTLE_TM_SERVER`.
 
 You may want to disable Amagama by setting :setting:`AMAGAMA_URL` to ``''`` if
-you are using Elasticsearch local TM, though the two will operate together.
+you are using Elasticsearch TM, though the two will operate together.
+
+
+.. _translation_memory#external_translation_memory:
+
+Elasticsearch based external TM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7.3
+
+By default this is configured to operate if external TM is available. To use it
+you will need to install `Elasticsearch 
+<https://www.elastic.co/products/elasticsearch>`_ and populate the initial
+translation memory using the :djadmin:`update_external_tmserver` command:
+
+.. code-block:: bash
+
+   (env) $ pootle update_external_tmserver af.po gl.tmx
+
+
+It is possible to have several Elasticsearch based external TM servers working
+at once, along with Elasticsearch based local TM servers. In order to do so
+just add new entries to :setting:`POOTLE_TM_SERVER`:
+
+.. code-block:: python
+
+    POOTLE_TM_SERVER = {
+
+        ...
+
+        'libreoffice': {
+            'ENGINE': 'pootle.core.search.backends.ElasticSearchBackend',
+            'HOST': 'localhost',
+            'PORT': 9200,
+            'INDEX_NAME': 'whatever',
+            'TM_NAME': 'LibreOffice 4.2',
+            'WEIGHT': 0.9,
+            'IS_LOCAL': False,
+            'MIN_SCORE': 'AUTO',
+        },
+    }
+
+Make sure ``IS_LOCAL`` is set to ``False`` and that ``INDEX_NAME`` is not used
+by any other TM server.
+
+Other values that you might want to tweak are ``TM_NAME`` (used to identify the
+TM in the translation editor) and ``WEIGHT`` which is used to change the
+relevance of the TM results in relation to other TM servers (valid values are
+between 0 and 1).
+
+To use them you will need to install
+`Elasticsearch <https://www.elastic.co/products/elasticsearch>`_ and populate
+them using the :djadmin:`update_external_tmserver` command:
+
+.. code-block:: bash
+
+   (env) $ pootle update_external_tmserver --tm=libreoffice af.po gl.tmx
+
+
+Note that the value passed to :option:`--tm` must match the ``INDEX_NAME``
+value in the server settings. Check :djadmin:`update_external_tmserver` for
+more options.
+
+Note that Pootle will not push new translations to these TM servers unless you
+explicitly use the :djadmin:`update_external_tmserver` command, giving you full
+control of which translations make into them.
+
+.. note:: Elasticsearch depends on Java. Note that some systems may ship with
+  OpenJDK, however `elasticsearch recommends using Oracle JDK
+  <https://www.elastic.co/guide/en/elasticsearch/reference/1.6/setup-service.html#_installing_the_oracle_jdk>`_.
+
+
+You may want to disable Amagama by setting :setting:`AMAGAMA_URL` to ``''`` if
+you are using Elasticsearch TM, though the two will operate together.
```

Comment:
This is repeating what is already said in local TM.  And the pragraph above.  Rather change structure then repeating the same stuff that someone will forget to maintain.